src/julia_gc.c

```diff
@@ -30,17 +30,20 @@
 
 #include "julia.h"
 #include "julia_gcext.h"
+#include "julia_gc.h"
 
 
 /****************************************************************************
 **
 **  Various options controlling special features of the Julia GC code follow
 */
 
-// if DISABLE_BIGVAL_TRACKING is defined, we don't track the location of
-// large bags; this speeds up some things, but may expose bugs in GAP code
-// which incorrectly holds pointers into bags over a GC.
-// #define DISABLE_BIGVAL_TRACKING
+// if SCAN_STACK_FOR_MPTRS_ONLY is defined, stack scanning will only
+// look for references to master pointers, but not bags themselves.
+// This should be safe, as GASMAN uses the same mechanism and is therefore
```

Comment:
Wouldn't this be more accurate?
```suggestion
// This should be safe, as GASMAN uses the same mechanism. It is also faster
// and avoids certain complicated issues that can lead to crashes, and is therefore
```